app/module/utils.go

```diff
@@ -0,0 +1,45 @@
+package module
+
+import (
+	"slices"
+	"sort"
+
+	sdkmodule "github.com/cosmos/cosmos-sdk/types/module"
+)
+
+const authName = "auth"
```

Comment:
nit: I wonder if you could just get that name from the auth module. Not that it should ever change

Reply:
Great call b363d9f3dfdeb1d390a052c9374986bbf517ad3e